### ISSUE TITLE
Issue installing with non localhost database server.

### ISSUE THREAD

**Issue by bweston92** [joomla/joomla-cms#17297]
### Steps to reproduce the issue

Attempt to install with non localhost.

- Goto installation
- Add non localhost database server config.
- Delete the temporary file.
- Progress to installation.
- Installation just stays still.

### Expected result

That after I have already deleted the file, the final installation should work flawlessly.

### Actual result

It just werls around and makes thousands of requests indefinitely as shown below.

![screenshot from 2017-07-27 09-30-37](https://user-images.githubusercontent.com/1764057/28661377-5b518104-72ae-11e7-8ec7-1d7d676a52e4.png)

### How I got it working

Whilst the progress bar was still making requests I updated the following file and it continues immediately.

Updated `/installation/model/database.php `

```
-                 if (!in_array($options->db_host, $localhost))
+                 if (false)
```

### System information (as much as possible)

- PHP 7.1
- Apache
- MySQL 5.7

**Comment by ghost** [comment]
can you please look if #17248 is same Issue and if so test Pull request #17248?

**Comment by zero-24** [comment]
This issue is fixed by #17248 please test.
